src/one_dragon_qt/widgets/download_card/launcher_download_card.py

```
@@ -0,0 +1,279 @@
+        :return:
+        """
+        if success:
+            # 下载成功，删除备份文件
+            self._cleanup_backup_launcher()
+
+            # 重置当前版本号以便下次检查更新
+            self.current_version = ""
+            # 更新标题栏版本号
+            try:
+                self.window().titleBar.setLauncherVersion(app_utils.get_launcher_version())
+            except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)